src/main/java/com/ait/lienzo/client/core/shape/TextNoWrap.java

```diff
@@ -0,0 +1,77 @@
+/*
+   Copyright (c) 2017 Ahome' Innovation Technologies. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+package com.ait.lienzo.client.core.shape;
+
+import com.ait.lienzo.client.core.Context2D;
+import com.ait.lienzo.client.core.types.BoundingBox;
+import com.ait.lienzo.shared.core.types.TextAlign;
+import com.ait.lienzo.shared.core.types.TextBaseLine;
+import com.ait.lienzo.shared.core.types.TextUnit;
+
+/**
+ * Text implementation for Canvas.
+ */
+public class TextNoWrap implements ITextWrapper
+{
+
+    protected final ITextWrapper.Supplier<String> textSupplier;
+    protected final ITextWrapper.Supplier<Double> fontSizeSupplier;
+    protected final ITextWrapper.Supplier<String> fontStyleSupplier;
+    protected final ITextWrapper.Supplier<String> fontFamilySupplier;
+    protected final ITextWrapper.Supplier<TextUnit> textUnitSupplier;
+    protected final ITextWrapper.Supplier<TextBaseLine> textBaseLineSupplier;
+    protected final ITextWrapper.Supplier<TextAlign> textAlignSupplier;
+
+    public TextNoWrap(final ITextWrapper.Supplier<String> textSupplier,
```

Comment:
It'll be convenient if there is a constructor that takes in a Text object and generate all the supplies for you, like in `noWrap()` in `Text`. Also applies to the other implementations of `ITextWrapper`.